node-graph/gcore/src/text.rs

```diff
@@ -29,3 +42,142 @@ impl From<TextAlign> for parley::Alignment {
 		}
 	}
 }
+
+#[derive(Clone, DynAny)]
+pub struct Typography {
+	pub layout: Layout<()>,
+	pub font_family: String,
+	pub color: Color,
+	pub stroke: Option<(Color, f64)>,
+}
+
+impl fmt::Debug for Typography {
+	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+		f.debug_struct("Typography")
+			.field("font_family", &self.font_family)
+			.field("color", &self.color)
+			.field("stroke", &self.stroke)
+			.finish()
+	}
+}
+
+impl PartialEq for Typography {
+	fn eq(&self, _other: &Self) -> bool {
+		true
+	}
+}
+
+impl Hash for Typography {
+	fn hash<H: Hasher>(&self, state: &mut H) {
+		self.layout.len().hash(state);
+	}
+}
+
+impl Typography {
+	pub fn to_vector(&self) -> Table<Vector> {
+		Table::new()
+	}
```

Comment:
Why did you add this method? This is a footgun waiting to happen. It would be very reasonable to expect a `to_vector` method to exist on Layout, but this just does nothing?